src/service/courseService.ts

```diff
@@ -44,11 +48,10 @@ class CourseService {
                 }
             });
             if (affectedRows === 0) {
-                throw new Error();
+                logger.error(errorEnums.ERR_INVALID_INPUT);
             }
             return successEnums.DELETE_SUCCESS;
```

Comment:
In linne no 51 you are logging error and then line 53 you are returning success. Correct here and if you see in other place also.